addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -735,19 +736,37 @@ blockquote {
             $-btn-primary-border-color: if($-btn-primary-border, o-color($-btn-primary-border), $-btn-primary-color);
             .btn-fill-primary {
                 @include button-variant($-btn-primary-color, $-btn-primary-border-color);
+                @if o-website-value('o-cc#{$index}-btn-primary-gradient') {
+                    background-image: o-website-value('o-cc#{$index}-btn-primary-gradient');
+                    &:hover {
+                        filter: invert(0.2);
+                    }
+                } @else {
+                    background-image: none;
+                }
```

Comment:
Added comment. This is actually needed when different `o_cc`s are nested.
E.g. if the section's `o_cc` defines gradient buttons, and the nested column's `o_cc` defines solid buttons, the parent's gradient would get painted instead of the defined solid color.